### DISCUSSION TITLE
weird favicon from 4/30/23

### DISCUSSION THREAD

**Discussion by polygoners1** [PenguinMod/PenguinMod-Home#117] [Q&A] [answered]
for probably like a few minuets on april 30th 2023 the favicon for penguinmod looked like this :
![april 30th favicon](https://github.com/user-attachments/assets/8f93ee36-41f6-4102-8ad9-fdfb4baa92e0)
and also here is a mockup of what it would look like to have the tab open (made by customizing the title and favicon.)
![Screenshot 2024-07-15 9 31 01 AM](https://github.com/user-attachments/assets/1b947203-9662-4b49-bad0-8a52579981de)
it was reverted a few minuets later in the "more stuffs 😋 " commit. why was this here?

**Comment by VyvaHart** [comment]
Hi @polygoners1 

From my point of view it’s possible that it was just a temporary change that the developer was testing. It’s also possible that it was a mistake that the developer accidentally committed and then fixed.

good luck!

**Reply by polygoners1** [reply]
oh, but i think i need full confirmation from a dev but this might be it!

**Comment by RedMan13** [comment, marked as answer]
that icon is the default sveltekit icon, what likely happened is between commits the icon file was in some way unavailable

**Reply by polygoners1** [reply]
Ohhhh ok. thx!
